demisto_sdk/commands/validate/file_validator.py

```diff
@@ -674,18 +680,43 @@ def run_all_validations_on_file(self, file_path: str, file_type: str = None) ->
                         'Layouts and Descriptions')
             self._is_valid = False
 
-    def validate_all_files(self):
+    def validate_all_files(self, skip_conf_json):
         print('Validating all files')
-        print('Validating conf.json')
-        conf_json_validator = ConfJsonValidator()
-        if not conf_json_validator.is_valid_conf_json():
-            self._is_valid = False
+
+        if not skip_conf_json:
+            print('Validating conf.json')
+            conf_json_validator = ConfJsonValidator()
+            if not conf_json_validator.is_valid_conf_json():
+                self._is_valid = False
```

Comment:
we have this on line 804, do we need this here too?

Reply:
This is not the same piece of code/use case. I don't see a reason to have them both, but I will not change this in this PR.
`validate_all_files ` != `validate_all_files_schema `